### PR TITLE
feat: RPM & DEB package building

### PR DESCRIPTION
**Pull request by DickPostma**
What are the main changes you did:
- Creating Debian and Redhat packages for easy deployment.
- 
how to test:
- Add the corresponding repository's

todo:
- [ ] updated docs for installation proces
- [ ] create cherry pick script 
- [ ] Create internal work document to build an stable cherry pick image.

### CODE REVIEW

**Review by mswertz — COMMENTED** [review]
when would a package be published to nexus? or will that be another PR? we could for now cherry pick a 'LTS' by hand. However, if we want to do rapid deploys like we now do but then using this then we should publish an 'experimental' package at every release.

**Comment by EleanorHyde-UMCG** [comment]
> Create a process What is a stable release ? @EleanorHyde-UMCG

Can you give a bit more context? Stable release of what?

**Comment by DickPostma** [comment]
> > Create a process What is a stable release ? @EleanorHyde-UMCG
> 
> Can you give a bit more context? Stable release of what?

Sorry @EleanorHyde-UMCG i was supposed to type this in the internal issue's. But we need to put up an process (as stated in the roadmap day) to cherry pick an stable release. Let's discuss this later on face2face.

**Comment by DickPostma** [comment]
$NEXUS_USER and $NEXUS_PASS are not available in the shell script anymore?
The ci-build image needs the rpm package.

**Comment by DickPostma** [comment]
Waiting on selecting a stable version before release.

**Comment by DickPostma** [comment]
I talked with @GeriekeBeen. We are gonna start with 10.79.2 as a first stable release.

File: docs/molgenis/run_systemd.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+
+# Install Molgenis emx2 as a system service
```
Comment (dtroelofsprins) [inline docs/molgenis/run_systemd.md:2]: In overige documentatie wordt MOLGENIS met hoofdletters geschreven. emx2 = EMX2 (al wordt dit in sommige delen van de documentatie ook weggelaten).

File: docs/molgenis/run_systemd.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
+
+# Install Molgenis emx2 as a system service 
+We Support redhat and ubuntu based installations.
```
Comment (dtroelofsprins) [inline docs/molgenis/run_systemd.md:3]: Support = support
Daarnaast wellicht Redhat en Ubuntu met hoofdletters? (ik weet niet hoe deze normaal gerefereerd worden).

File: docs/molgenis/run_systemd.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+
+## Java 
+
+Molgenis-emx2 runs on java > 17
```
Comment (dtroelofsprins) [inline docs/molgenis/run_systemd.md:8]: idem voor MOLGENIS EMX2. 
Daarnaast staat in de installation guide: Install java (we use adopt [OpenJDK 17](https://adoptium.net/)). Ik draai ook op 17.

File: docs/molgenis/run_systemd.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+sudo yum install java-21-openjdk-headless
+
+```
+<!-- tabs:end -->
+
+
+## Postgresql 
+
+
+Molgenis-emx2 depends on Posgresql 15
```
Comment (dtroelofsprins) [inline docs/molgenis/run_systemd.md:28]: MOLGENIS EMX2
Posgresql bevat een typefout

File: docs/molgenis/run_systemd.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+
+
+
+## Application emx2 
+
+Molgenis is running port 8080. 
+* Open on http://ip.or.host.of.server:8080
+
+In most cases you can proxy_pass traffic to molgenis-emx2 with an (hard of software) loadbalancer to the molgenis-emx2 (port 8080) endpoind.
```
Comment (dtroelofsprins) [inline docs/molgenis/run_systemd.md:147]: an = a
of = or
endpoind = endpoint

File: docs/molgenis/run_systemd.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+Or you can use nginx to proxypass traffic.
+
+A example of nginx proxy pass:
```
Comment (dtroelofsprins) [inline docs/molgenis/run_systemd.md:150]: A = An

**Review by dtroelofsprins — CHANGES_REQUESTED** [review]
Only reviewed the run_systemd.md. If the command line in this doc is OK I cannot judge as I didn't test this. Only had some textual comments.

**Review by Fernananas — APPROVED** [review]
Tested run_systemd.md for ubuntu server. Result; molgenis server was installed and website could be reached. OK

File: docs/molgenis/run_systemd.md [hunk 7]
```diff
@@ -0,0 +1,199 @@
+
+# Install Molgenis emx2 as a system service
```
Comment (dtroelofsprins) [inline docs/molgenis/run_systemd.md:2]: Wellicht hier ook nog even naar MOLGENIS EMX2 aanpassen(?).

**Pull request merged**